docs/source/installation.rst

```diff
@@ -350,20 +350,36 @@ The building command depends on the architecture of the Python interpreter,
 The above commands assume that you have the Python installation folder in your
 PATH environment variable.
```

Comment:
maybe add a link how to modify Windows env variables (this is a very common topic, so perhaps somewhere on Microsoft webpage), and what a common path could look like (note this may differ in admin and in restricted user install)

Reply:
added this